components/x-gift-article/src/v2/AdvancedSharingOptions.jsx

```diff
@@ -43,12 +52,19 @@ export const AdvancedSharingOptions = ({ shareType, actions, showHighlightsCheck
 								value={ShareType.gift}
 								checked={shareType === ShareType.gift}
 								onChange={onValueChange}
+								disabled={!giftCredits}
 							/>
 							<span className="o-forms-input__label">One person</span>
 						</label>
 					</span>
 				</span>
 			</div>
+			{(!giftCredits || !enterpriseHasCredits) && (
+				<NoCreditAlert>
+					One of the non-subscriber choices is not available because you’ve run out of credits. Please use the
+					other option.
```

Comment:
**question:** why is "other option" on a new line?

Reply:
Prettier formatting is to blame here, once the code exceeds a certain point the formatter kicks in.